pkg/collector/corechecks/cluster/kubernetesapiserver/events_common.go

```diff
@@ -75,14 +75,33 @@ func getInvolvedObjectTags(involvedObject v1.ObjectReference) []string {
 	return tags
 }
 
+const (
+	podKind  = "Pod"
+	nodeKind = "Node"
+)
+
 func getEventHostInfo(clusterName string, ev *v1.Event) eventHostInfo {
+	return getEventHostInfoImpl(getHostProviderID, clusterName, ev)
```

Comment:
Why do we need two functions? If it's only for unit testing, we should have a comment.

Reply:
it is for unit-test. I didn't wanted to call the code that retrieve the host info